rfcs/README.md

```diff
@@ -2,14 +2,58 @@
 
+## Why RFCs?
+
+The RFC process is intended to bring a few benefits:
+
+- More efficent and durable decision making
+  - **Efficent** as we can all engage and review on our own time, rather than locking X engineers in a room for hours
+  - **Durable** as we give more time for decisions to be reviewed, commented, and considered. Rather than making a decision
+    after being locked in a room for an hour with X other engineers.
+- A more inclusive review proces
+  - Supporting both quick and deep thinkers
+  - Allowing proposals and comments from contributors
+- Capturing decisions, allowing:
+  - Us to review recent decisions and build ontop of them
+  - Others to join the project more easily
+
+The React RFC Process sums it up nicely[1]
+
+> The RFC process is a great opportunity to get more eyeballs on your proposal before it becomes a part of a released version. Quite often, even proposals that seem "obvious" can be significantly improved once a wider group of interested people have a chance to weigh in.
+>
+> The RFC process can also be helpful to encourage discussions about a proposed feature as it is being designed, and incorporate important constraints into the design while it's easier to change, before the design has been fully implemented.
+
 ## RFC process
 
-### Creation
+### 1. Creation
+
+- Start by making a copy of [TEMPLATE.md](./TEMPLATE.md) under an appropriate area sub-folder and name. (While we're still experimenting with the RFC process, feel free to create a new area sub-folder if you don't think an appropriate one exists yet.)
+- Write the RFC
+- Submit a PR and label it with **Type: RFC**.
+
+### 2. Review
 
-Start by making a copy of [TEMPLATE.md](./TEMPLATE.md) under an appropriate area sub-folder and name. (While we're still experimenting with the RFC process, feel free to create a new area sub-folder if you don't think an appropriate one exists yet.)
+A key purpose of the review phase is to seek feedback to help improve your idea. You the author are ultimately accountable for the proposal and it's impact. Reviewers can, through comments, help you make the best decision you can. Throughout this review and process it is important that we strive to avoid both [Design by Committee](https://en.wikipedia.org/wiki/Design_by_committee) and [Diffusion of Responsibility](https://en.wikipedia.org/wiki/Diffusion_of_responsibility).
 
-Once you're done writing, make a PR and label it with **Type: RFC**.
+- Seek support and integrate feedback.
+  - It is the responsibility of the author(s) to read, consider, and respond to all feedback in a timely manner. This does not mean all feedback must be integrated into the proposal, but it must be considered.
+  - Tip: If you want to good engagement on your RFC, reserve time to respond to comments quickly.
 
-### Implementation
+Once the RFC has been open for **at least 2 days**, and recieved sufficent comments, the author can begin seeking approval.
```

Comment:
**suggestion:**
- Create Draft PR
- gather initial comments and incorporate suggestions if applicable
- promote  PR to "Ready for review" aka ready for seeking approval

Reply:
Good suggestion. Maybe a label "Ready for final review". 

I was leaving it a little open and just saying "seeking approval" as there likely will be an aspect of actively going out and asking for approval. Tho having a label to mark it ready wouldn't change this and would only help.

Reply:
yeah, I mean label === PR Draft , no need for yet another label ? :D

Reply:
Ohh, ok. Yeah spinning up an RFC as a Draft would be fine and a good way to get early feedback.

I don't think we should require all RFCs to start as Draft. Just like a normal PR folks should use the Draft state to communicate that this is not ready for broad feedback. Also, just like a normal PR it's totally fine to reach out to specific people and ask them to weigh in on the PR early. 

I'll add a line talking about this.

I do think we should just assume that a non-draft PR is ready for broad review.